build/features.py

```diff
@@ -994,180 +1000,85 @@ def configure(self, build, conf):
             # build.env.Append(LINKFLAGS = '/OPT:NOWIN98')
             # ALBERT: NOWIN98 is not supported in MSVC 2010.
 
-            if build.build_is_debug:
-                self.status = "Disabled (debug build)"
-            
             # http://msdn.microsoft.com/en-us/library/59a3b321.aspx
             # In general, you should pick /O2 over /Ox
-            elif optimize_level >= 1:
-                self.status = "Enabled -- Maximize Speed (/O2)"
-                build.env.Append(CCFLAGS='/O2')
-            # elif optimize_level >= 2:
-            #    self.status = "Enabled -- Maximum Optimizations (/Ox)"
-            #    build.env.Append(CCFLAGS = '/Ox')
+            build.env.Append(CCFLAGS='/O2')
+
+            # Historicaly our release packages are built with optimize=9.
+            if optimize_level == 'portable' or optimize_level == '' or optimize_level == 9:
```

Comment:
SCons.ARGUMENTS contains a plain old Python str. Here's a debug trace from inside the Optimize feature:
```Python
value = SCons.ARGUMENTS.get('optimize', '')
import pdb
pdb.set_trace()
(Pdb) type(value)
<type 'str'>
(Pdb) value == '9'
True
(Pdb) value == 9
False
```

If it works, something else is making it work :).